eclass/qt5-build.eclass

```diff
@@ -58,6 +58,27 @@ esac
 # for tests you should proceed with setting VIRTUALX_REQUIRED=test.
 : ${VIRTUALX_REQUIRED:=manual}
 
+# @ECLASS-VARIABLE: QT5_DOCS
+# @DESCRIPTION:
+# List of docs tarballs that belong to a package. By default set to false.
+: ${QT5_DOCS:=false}
```

Comment:
so we're not defaulting to `$PN`?

Reply:
Since we have many packages without docs, and then several diverging from `$PN`, I'd not make it a default. It's mostly a first-time setup issue anyway.

Reply:
ok makes sense